packages/rancher-monitoring/rancher-service-metrics/charts/templates/servicemonitor.yaml

```diff
@@ -1,6 +1,9 @@
 {{- $chartLabel := .Values.chartLabel }}
 {{- if eq $chartLabel "" }}
-{{- $chartLabel = (index (lookup "apps/v1" "Deployment" "cattle-system" "rancher").metadata.labels "chart") }}
+{{- $rancher := (lookup "apps/v1" "Deployment" "cattle-system" "rancher") }}
+{{- if $rancher -}}
+{{- $chartLabel = (index $rancher.metadata.labels "chart") }}
+{{- end }}
```

Comment:
cc: @rmweir, I caught this after trying to deploy Monitoring. Seems to be a bug we missed in https://github.com/rancher/charts/pull/1712

Reply:
Fixed in https://github.com/rancher/charts/pull/1768/commits/4f04899ceec6190abe5abbe8081a8f2d6bf88c8f, dropped from this PR.